### PR TITLE
Adding HelloWin Program

### PR DESCRIPTION
**Pull request by SangameshwarGudali**
Adding WinDev (HelloWin) Programme.

### CODE REVIEW

**Comment by Shivkumar13** [comment]
@SangameshwarGudali Thanks for PR, LGTM!

File: HelloWinApp [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    WNDCLASSEX wndEx;
+    MSG msg;
+
+    ZeroMemory(&wndEx, sizeof(WNDCLASSEX));
+    ZeroMemory(&msg, sizeof(MSG));
+
+    hBrush = (HBRUSH)GetStockObject(WHITE_BRUSH);
+    assert(hBrush);
+
+    hIcon = LoadIconA(NULL, IDI_APPLICATION);
```
Comment (Shivkumar13) [inline HelloWinApp]:
```suggestion
    hIcon = LoadIcon(NULL, IDI_APPLICATION);
```

Comment (Shivkumar13) [reply HelloWinApp]: Could you change this so that it is same as Line 33

Comment (SangameshwarGudali) [reply HelloWinApp]: correction done.

**Comment by Shivkumar13** [comment]
LGTM!

**Pull request merged**